src/js/components/bulkDownload/sharedFilters/DefCodeFilter.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from "prop-types";
+
+import { CheckCircle } from 'components/sharedComponents/icons/Icons';
+import DEFCheckboxTreeDownload from 'components/bulkDownload/DEFCheckboxTreeDownload';
+
+const DefCodeFilter = ({ type }) => {
+    const icon = (
```

Comment:
This might be a question for design, not sure is we want a green checkmark when no filters are selected although, but they are optional. Looking at the other forms it does not look like we have optional fields to know how to handle this.